ASE2019-red/thingy-client-red
# Docker setup

Testing:
1. Build & start the container with `docker-compose up -d --build` (make sure backend is running on localhost:8000)
2. Check if http://localhost:8080/ is accessible and works as expected
